kentik_api_library/kentik_api/public/custom_dimension.py

```diff
@@ -132,7 +132,8 @@ def __init__(
         company_id: Optional[ID] = None,
     ) -> None:
         # read-write
-        self.name = name  # must start with c_ and be unique even against deleted dimensions (deleted names are retained for 1 year)
+        self.name = name  # must start with c_ and be unique even against deleted dimensions
```

Comment:
Multi-line comments are typically placed above the statement. Right now  it looks like`# (deleted names are retained for 1 year)` line applies to `self.display_name = display_name`
```suggestion
        # must start with c_ and be unique even against deleted dimensions
        # (deleted names are retained for 1 year)
        self.name = name  
```

Reply:
Done